### PR TITLE
ID53 - Cleaned up form, added sensitive data guide

### PR DESCRIPTION
**Pull request by itarpeh**
-Bootstrap formatting 
-Link to Umich sensitive data guide

### CODE REVIEW

**Comment by gsilver** [comment]
LGTM!

A few notes I left in the code review should be addressed.

A few details that we can address after the merge:

1. Reset state when changing choices. For example - if I choose **App**, then select options including DB options, then change my mind and choose **Simple Website**, the DB options block is still displayed
2. Adding back your styles.

Also - it would be good to not check in things that are just by products, like .DS_Store, etc. To do so:

From terminal:

1. cd ~
2. touch .gitignore
3. brackets .gitignore
4. add, one per line

.DS_Store
node_modules

this way they will not show up as files to add and commit and you can keep using `git add .`

File: src/views/order.html [hunk 1]
```diff
@@ -1,66 +1,182 @@
...
+        </div>-->
+        <fieldset class="form-group">
+            <legend>Additional Information</legend>
+            
+            <div id="url">
+              <!--<div ng-show="orderModel.url !==null">
+                URL : {{orderModel.url}}
+              </div>-->
+              <label for="url-input">Choose a URL</label>
+              <input type="url" ng-model=orderModel.url class="form-control" placeholder="Enter url">
```
Comment (gsilver) [inline src/views/order.html]: **orderModel.url** needs to be quoted. Also type="url" is legit, but they are only going to be entering a fragment, so we are not going to be validating as a **url** and Angular seems to void (goes from null to nada when you start typing) so we are better of with type="text"

File: src/views/order.html [hunk 2]
```diff
@@ -1,66 +1,182 @@
...
+              </div>
+            </div>
+            
+            <!--<div ng-show="orderModel.shortcode !==null">
+              URL : {{orderModel.shortcode}}
+            </div>-->
+            
+            <div id="shortcode" style="margin-top:50px;">
+              <label for="shortcode-input">Enter a shortcode</label>
+              <input type="text" ng-model=orderModel.shortcode class="form-control" placeholder="Enter shortcode">
```
Comment (gsilver) [inline src/views/order.html]: **orderModel.shortcode** needs to be quoted.

File: src/views/order.html [hunk 3]
```diff
@@ -1,66 +1,182 @@
...
 
-    <div id="itar">
-      Sensitive data?
-      <input type="radio" ng-model="orderModel.itar" value="no">No
-      <input type="radio" ng-model="orderModel.itar" value="yes">Yes
-    </div>
-    <div id="url">
-      <div ng-show="orderModel.url !==null">
-        URL : {{orderModel.url}}
+      <!-- this sets the app choices if set -->
```
Comment (gsilver) [inline src/views/order.html]: Let's remove commented out code through out too.

File: src/views/order.html [hunk 4]
```diff
@@ -1,66 +1,182 @@
...
+              <input type="text" ng-model=orderModel.shortcode class="form-control" placeholder="Enter shortcode">
+              <small id="shortcodeHelp" class="form-text text-muted">Lorem ipsum dolor sit amet, consectetur adipiscing.</small>
+            </div>
+        </fieldset>
+      </div>
+        
     </div>
-  </div>
-  </div
+</div>
```
Comment (itarpeh) [inline src/views/order.html]: test comment

**Pull request merged**